### ISSUE TITLE
Check debian-live-config project

### ISSUE THREAD

**Issue by probonopd** [probonopd/live-build-travis#1]
https://github.com/nodiscc/debian-live-config

**Comment by nodiscc** [comment]
Hi, I found this issue by searching for my handle on Github :) I am very interested in getting live-build to work in CI.

I made a few attempts at running the build in Gitlab CI (from the mirror at https://gitlab.com/nodiscc/debian-live-config - I did not want to use proprietary CI hence no travis or GH actions), I came to the following conclusions:
 - disk space requirements for the build are huge (~13GB cache + 2GB ISO) and well above what any free CI provider will allow
 - live-build must have effective root access because it manipulates `/dev`, `/proc` filesystems in complex ways. Hence free shared/Docker-based runners will fail. I imagine the same is true for Travis.
 - special attention must be given to caching configuration (else you wiill end up re-downloading all packages every time)

So the plan is the following:
 - setup a private gitlab-runner machine, running she [`shell` executor](https://docs.gitlab.com/runner/executors/) with maximum privileges (including sudoers rule to run the build as root)
 - register the runner as project-specific runner on the Gitlab project
 - configure `.gitlab-ci.yml` to only run builds on protected branches (else anyone sending a merge request would effectively have root on the runner)
 - make it work :)
